### DISCUSSION TITLE
Adding the ability to disable update reminders

### DISCUSSION THREAD

**Discussion by Nikon4ik0509** [seerge/g-helper#3002] [Ideas]
Thank you for your wonderful app! But please add the ability to disable the update reminder bar in the next update :) This devilish red bar annoys me;)

**Comment by seerge** [comment]
@Nikon4ik0509 hello. 

Sorry to read that you find update label "devilish".  The best way to make it grey is to actually update your app as this is always recommended.

Thanks.

**Reply by Parkss1181** [reply]
@seerge is it possible I can run this on my gigabyte mobo on my desktop? Or do you know if an alternative software for gigabyte control center? Cause it is just absolutely horrible. I have installed g helper on my Asus laptop and it's amazing. If you could provide any info on this I would really appreciate it. Thanks

**Reply by seerge** [reply]
@Parkss1181 you can run it on desktop PC, but it won't do there anything besides allowing you to set settings for an Asus mouse if you have one

**Reply by Parkss1181** [reply]
> @Parkss1181 you can run it on desktop PC, but it won't do there anything besides allowing you to set settings for an Asus mouse if you have one

Is there any softwares similar that I could possibly use or you can create like g helper for a gigabyte mother board? Gigabyte control center is just so bad.

**Reply by seerge** [reply]
No, I can't "create" g-helper for gigabyte motherboards :) if you need RGB controls most of PC people seem to use either OpenRGB or SignalRGB, and [Rem0o FanControl](https://github.com/Rem0o/FanControl.Releases) for fans

**Reply by Parkss1181** [reply]
> No, I can't "create" g-helper for gigabyte motherboards :) if you need RGB controls most of PC people seem to use either OpenRGB or SignalRGB, and [Rem0o FanControl](https://github.com/Rem0o/FanControl.Releases) for fans

I actually have been running signal RGB but it gets buggy since GCC tries to override it even when I turn off the RGB controls from the gcc control center.
